src/doc/DocHighlightDialog.js

```diff
@@ -47,10 +47,20 @@ class DocHighlightDialog extends AnnotationDialog {
             ]);
             util.showElement(highlightLabelEl);
 
+            const btns = this.highlightDialogEl.querySelectorAll('button');
+            Array.prototype.forEach.call(btns, (btn) => {
+                btn.classList.remove(constants.CLASS_DISABLED);
+            });
+
             // Only reposition if mouse is still hovering over the dialog and not mobile
             if (!this.isMobile) {
                 this.position();
             }
+        } else if (firstAnnotation.isPending && firstAnnotation.type === 'highlight' && annotations.length === 1) {
```

Comment:
Is this temporary? I'm thinking that when these buttons are in react they will be set to be disabled by the parent component?

Reply:
Yup! This will be removed once the entire dialog is moved into react. The plain highlight dialog is a little weird while in this intermediate state

Reply:
The enable/disable methods were causing some problems while I was integrating the AnnotationList component into the dialog so I removed it. I don't expect to get to handling this case in the next few components that i'll create so I added this.